src/data/faq.json

```diff
@@ -2063,12 +2073,40 @@
+                }
+            ],
             "S": [
                 {
                     "term": "Signature",
                     "description": "A digital signature is a signature that is created by a crytographic function from a digital document. Digital signatures provide the following properties: <ul><li>Non-repudiation: The issuer cannot retrospectively deny signing the document.</li><li>Integrity: Altering the document invalidates the signature.</li><li>Authenticity: It is clear who signed the document.</li></ul> <br/><br/> As a consequence, the signature must fulfill the following requirements: <br/><ul><li>The signature must be computationally verifiable </li><li>Verifying the signature must be easy - forging a valid signature must be hard</li><li>The signature is only valid for the signed document and no other documents </li><li>The issuer must not be able to deny signing the document.</li></ul>"
                 }
-            ]
+            ],
+            "T": [
+                {
+                    "term": "Test certificate",
+                    "description": "A test certificate is a => digital certificate issued to a person who has completed a COVID-19 test with a negative result."
+                }
+            ],
+            "V": [
+                {
+                    "term": "Validity of an EU digital COVID certificate",
+                    "description": "Basically, we distinguish between <ul><li>the validity of the digital certificate (the technical validity) and</li><li>the validity of the vaccination protection (the effective validity).</li></ul>The digital certificate has an issue date and a technical expiration date (so does your passport and PA). The validity of the certificate ends 1 year after issuance. <br/><br/> <b>Vaccination Certificate</b> <br/><br/>In the certificate data is transported, in this case information about vaccination. This vaccination took place on a date X and is valid from then (according to the current state of scientific knowledge) e.g. in Germany 1 year. After that the vaccination protection has expired. We call the duration of the vaccination protection 'effective (vaccination) validity'. So the effective validity of the vaccination counts from the last vaccination date (date X).  <br/><br/> A certificate is valid until either the technical validity or the effective validity expires."
+                },
+                {
+                    "term": "Vaccination certificate",
+                    "description": "The vaccination certificate is a => digital certificate issued to a person who has received a vaccination with a vaccine approved in the EU. Each vaccination certificate documents exactly one vaccination event (one dose). <br/><br/>The vaccination certificate represents a digital form of a => proof of vaccination."

```

Comment:
issued to --> issued for

Reply:
Done via https://github.com/corona-warn-app/cwa-website/commit/529b7e25f68173e5ffeabe6c3968492bb6d8f967.